### PR TITLE
Expose more istio attribute options in profile controller

### PR DESCRIPTION
**Pull request by Jeffwan**
Address Issue https://github.com/kubeflow/kubeflow/issues/4746

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/4747)

### CODE REVIEW

File: components/profile-controller/controllers/profile_controller.go [hunk 1]
```diff
@@ -377,6 +379,14 @@ func (r *ProfileReconciler) updateIstioRbac(profileIns *profilev1.Profile) error
 			}
 		}
 	}
+
+	// Check all options here https://istio.io/docs/reference/config/policy-and-telemetry/attribute-vocabulary/
+	var propertyKey string
+	if r.AttributeType == "map" {
```
Comment (jlewi) [inline components/profile-controller/controllers/profile_controller.go]: Would it possible to write a unittest to ensure we generate the expected ServiceRoleBinding?

**Comment by jlewi** [comment]
@Jeffwan so just to double check; it looks like you added some command line arguments. If those flags aren't set and the default values are used the behavior matches the current behavior?

File: components/profile-controller/controllers/profile_controller.go (hunk 1 above)
Comment (Jeffwan) [reply components/profile-controller/controllers/profile_controller.go]: Definitely. I should. I will update the PR later.

**Comment by kunmingg** [comment]
Thanks for the update!

**Comment by Jeffwan** [comment]
I will close this PR at this moment and I move to a different solution without profile controller change. https://github.com/kubeflow/manifests/pull/908  I will reopen whenever there's other similar requirement.

**Pull request closed without merging**